input/pagecontent/aucdi.md

```diff
@@ -85,22 +86,28 @@ Column attribute descriptions are as follows:
     <td></td>
   </tr>
   <tr>
-    <td rowspan="5">Procedure completed event</td>
+    <td colspan="2">Last updated</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+  <tr>
+    <td rowspan="5">Procedure completed</td>
```

Comment:
It is actually "Procedure completed event"; sorry - I think I changed this one. Please revert.

Reply:
At this point it's 50/50. I'll make the change and we'll re-evaluate when AUCDI R1 is published.